assets/javascripts/discourse/templates/components/group-assigned-filter.hbs

```diff
@@ -13,6 +13,19 @@
       {{filter.assignments_count}}
     </div>
   {{/link-to}}
+{{else if group-name}}
```

Comment:
We prefer `camelCase` for variables in Javascript/HBS. The exception is component names which should have a dash.